hyperspy/drawing/utils.py

```diff
@@ -315,7 +315,7 @@ def _make_overlap_plot(spectra, ax, color="blue", line_style='-'):
         x_axis = spectrum.axes_manager.signal_axes[0]
         ax.plot(x_axis.axis, spectrum.data, color=color, ls=line_style)
     if len(spectra) > 1:
```

Comment:
Although I have to admit I don't understand why the xlabel is only set `if len(spectra) > 1`...